src/languages/es.ts

```diff
@@ -138,7 +138,7 @@ export default {
         magicCode: 'Código mágico',
         twoFactorCode: 'Autenticación de dos factores',
         workspaces: 'Espacios de trabajo',
-        chats: 'Chats',
+        inbox: 'Bandeja de entrada',
```

Comment:
This text overflows https://github.com/Expensify/App/pull/42772#issuecomment-2156032649

Reply:
@s77rt do we have a fix for that yet?